.github/workflows/ruby.yml

```diff
@@ -24,8 +24,13 @@ jobs:
         env:
           BUNDLE_GEMS__CONTRIBSYS__COM: ${{ secrets.BUNDLE_GEMS__CONTRIBSYS__COM }}
 
-      - name: Install yarn dependencies
-        run: yarn --ignore-engines
+      - name: Set up node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.20.0
+
+      - name: Install yarn dependencies and precompile assets
+        run: bundle exec rails assets:precompile
```

Comment:
Should lines 60-64 get a similar edit? I don't understand how this node/yarn stuff works.

Reply:
No it shouldn't be needed. For some reason the specs for some repos complained about assets not being available so I added the precompile step which fixes that. It was only rspec that had that issue.